android/app/build.gradle

```diff
@@ -47,10 +48,12 @@ android {
         applicationId "com.example.star_movie_app"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion 19
```

Comment:
keep using flutter.minSdkVersion variable, but change its value to 19 (or even can use 21)

if u will have some free time - u can read [this](https://developer.android.com/about/versions) doc and try to remember/note somewhere 2-3 new features from lollipop to android 13, that can be useful for u

Reply:
Fixed.